Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -595,6 +595,76 @@ Messages::WebDriverClient::NewWindowResponse WebDriverConnection::new_window(Jso
     return result;
 }
 
+// 11.6 Switch To Frame, https://w3c.github.io/webdriver/#dfn-switch-to-frame
+Messages::WebDriverClient::SwitchToFrameResponse WebDriverConnection::switch_to_frame()
+{
```

Comment:
If you're going to make stub methods like this, please make them write a debug message so that it's clear we're trying to run something that's unimplemented. eg:

```c++
    dbgln("FIXME: WebDriverConnection::switch_to_frame()");
```

Same for the other methods here.